tuples.go

```diff
@@ -34,6 +34,8 @@ const (
 const (
 	// default page size open fga max is 100
 	defaultPageSize = 100
+	// Writes default size is 10
+	defaultSize = 10
```

Comment:
name this variable to be more accurate about what it's for and provide reference for future readers of the code

```suggestion
	// maxWrites is the maximum number of Writes and Deletes supported by the OpenFGA transactional write api
	// see https://openfga.dev/docs/interacting/transactional-writes for more details
	maxWrites = 10
```